src/ui/components/Timeline/Capsule.module.css

```diff
@@ -49,3 +50,21 @@
   width: 1.5rem;
   height: 1.5rem;
 }
+
+.SlowLoadingPopout {
```

Comment:
Is the idea behind CamelCase for the class names that they will be the exact same token as the component they are styling?